### ISSUE TITLE
Image upload does not overwrite original

### ISSUE THREAD

**Issue by arj0019** [typst/typst#1262]
### Description

When uploading an image (~.png) with the same file name as one that is already in the project, the new image does not overwrite the original image.

The web app presents a warning message stating that the file will be overwritten, which is currently incorrect.

I have not checked the behavior for other file types.

### Browsers

Chrome

### OS

Windows

**Comment by thygesteffensen** [comment]
Hello @reknih ,

Since this is closed and from this line from the release notes of 0.4.0
>  Fixed many bugs regarding file upload and more

I assume this issue was fixed.

However, I still experience that files are not overwriting when pressing "Overwrite"/"Upload file" in the pop-up dialogue.
This is the case using the upload button and the drag-and-drop feature. 

Deleting the file and then uploading results in the correct version being present in my project :) 

I'm running Ubuntu 22.04 and using Microsoft Edge (114.0.1823.37 (Official build) (64-bit)). I did a hard reload and assume I'm using the latest version of the web app - I experienced this today.
